tests/modalTest.spec.ts

```diff
@@ -0,0 +1,16 @@
+import { test } from "./fixtures";
+import { expect } from "@playwright/test";
+
+test("Facebook button work", async ({ page }) => {
+  await page.getByTestId("King button").click();
+  await page.getByAltText("Facebook").click();
+  const pagePromise = page.waitForEvent("popup");
+  const newTab = await pagePromise;
+  await newTab.waitForLoadState();
```

Comment:
this should not be used even though it does the job because it can only be used once for the entire test. In case you want to make the test do something else as well it will create a few problems.